controllers/storagecluster/cephcluster.go

```diff
@@ -103,6 +109,13 @@ func (obj *ocsCephCluster) ensureCreated(r *StorageClusterReconciler, sc *ocsv1.
 		return reconcile.Result{}, fmt.Errorf("'StorageDeviceSets' should not be initialized in an external CephCluster")
 	}
 
+	err = r.setReadAffinity(sc, !sc.Spec.ExternalStorage.Enable)
```

Comment:
why do we need to pass a boolean again when we are already passing the storagecluster? we can read the value from storagecluster directly in the func.